cob_description/urdf/sensors/realsense_d435.urdf.xacro

```diff
@@ -2,7 +2,8 @@
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
 
   <xacro:include filename="$(find cob_description)/urdf/common.xacro" />
-  <xacro:include filename="$(find cob_description)/urdf/sensors/realsense_d435.gazebo.xacro" />
+  <!--xacro:include filename="$(find cob_description)/urdf/sensors/realsense_d435.gazebo.xacro" /-->
```

Comment:
If the new plugin works, let's not add two plugins, but just one. Or is there a reason for keeping the old one?

Reply:
Yes, will only leave one single `d435`. Will do that in a final commit after everything's tested.

Reply:
it's not decided whether we even update this xacro in cob_description as it requires code from an unreleased repo...